Xamarin.Forms.Maps.UWP/MapRenderer.cs

```diff
@@ -235,6 +240,8 @@ void UpdateVisibleRegion()
 
 		void LoadUserPosition(Geocoordinate userCoordinate, bool center)
 		{
+			if (Control == null || Element == null) return;
```

Comment:
Sorry reviewing this again, can we do the same we did here on 
so we can keep the code as it is there.. 

`UpdateIsShowingUser`